docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+Thank you very much for your interest in contributing to our project! As a public service company, we want to shape a product that better matches our user needs and desires. Ideas or direct contributions are therefore warmly welcome and will be considered with great care, provided they fulfill a few requirements listed in this document. Please read it first before you decide to contribute.
+
+## Purpose
+
+Please follow the present contributing guidelines so that we can efficiently consider your proposal. You should also read or our [code of conduct](CODE_OF_CONDUCT.md), providing a few guidelines to keep interactions as respectful as possible.
```

Comment:
```suggestion
Please follow the present contributing guidelines so that we can efficiently consider your proposal. You should also read our [code of conduct](CODE_OF_CONDUCT.md), providing a few guidelines to keep interactions as respectful as possible.
```